scripts/mostReadAndWatchedCollectionData.js

```diff
@@ -13,7 +13,8 @@ const badServices = [
   'ukrainianRu',
 ];
 
-const servicesWithVariants = ['zhongwen', 'serbian'];
+// Who knows?
+const servicesWithVariants = ['zhongwen', 'serbian', 'uzbek'];
```

Comment:
Added this here since 'servicesWithVariants' came up in a search. I need to better understand the knock on effect here.